modules/op-release-notes-1-8.adoc

```diff
@@ -26,14 +26,20 @@ include::snippets/technology-preview.adoc[]
 
 * Support for the `tekton.dev/v1alpha1` API version, which was deprecated in {pipelines-title} GA 1.6, is planned to be removed in the upcoming {pipelines-title} GA 1.9 release.
 +
-This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. The `operator.tekton.dev/v1alpha1` API version is not deprecated and still used by the Operator.
+This change affects the pipeline component, which includes the `TaskRun`, `PipelineRun`, `Task`, `Pipeline`, and similar `tekton.dev/v1alpha1` resources. Update existing resources to use `apiVersion: tekton.dev/v1beta1` instead.
```

Comment:
nit: I think we can't link to outside document, right ?
The migration is document here : https://tekton.dev/docs/pipelines/migrating-v1alpha1-to-v1beta1/. I wonder if we should have one for docs.openshift.com as well 🤔

Reply:
Actually, modules like this **can** link to outside documents, just not to other files in the docs. It's unclear whether you're proposing to link to that topic and/or re-document the tekton topic in the OpenShift docs. @sounix000, there might be a doc task here.